operators/osdk-getting-started.adoc

```diff
@@ -0,0 +1,58 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+[id='getting-started-osdk']
+= Getting started with the Operator SDK
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: osdk-getting-started
+
+toc::[]
+
+{nbsp} +
+This guide walks through an example of building a simple Memcached Operator
+(`memcached-operator`) using tools and libraries provided by the Operator SDK,
+and shows how to manage the new Operator's lifecycle from installation through
+updating to a new version.
+
+As an administrator of a Kubernetes-based cluster, you can accomplish this using
+two centerpieces of the Operator Framework:
+
+Operator SDK:: Assists developers in bootstrapping and building an Operator
+based on their expertise without requiring knowledge of Kubernetes API
+complexities.
+
+Operator Lifecycle Manager (OLM):: Helps Operator users install, update, and
+generally manage the lifecycle of all Operators and their associated services
+running across their clusters.
+
+include::modules/about-operator-sdk.adoc[leveloffset=+1]
+include::modules/installing-operator-sdk-cli.adoc[leveloffset=+1]
+include::modules/building-memcached-operator-using-osdk.adoc[leveloffset=+1]
+include::modules/managing-memcached-operator-using-olm.adoc[leveloffset=+1]
+
+== Getting involved
+
+This guide provides an effective demonstration of the value of the Operator
+Framework for building and managing Operators, but this is much more left out in
+the interest of brevity. The Operator Framework and its components are open
+source, so visit each project individually and learn what else you can do:
+
+link:https://github.com/operator-framework[*github.com/operator-framework*]
+
+If you want to discuss your experience, have questions, or want to get involved,
+join the
+link:https://groups.google.com/forum/#!forum/operator-framework[Operator
+Framework mailing list].
+
+== Appendices
+
+[discrete]
+include::modules/operator-project-staffolding-layout.adoc[leveloffset=+2]
```

Comment:
Actually, the `version` folder not list in the `Operator project scaffolding layout` part. I think we should list it.

Reply:
I've added `version` to the table:

http://file.rdu.redhat.com/~adellape/083018/osdk_gs/operators/osdk-getting-started.html#appendices

@robszumski Better wording here?